Forge.TreeWalker.UnitTests/test/ActionsCallbacksAndValidator/TreeSchemaValidator.cs

```diff
@@ -0,0 +1,277 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="TreeSchemaValidator.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// <summary>
+//     The TreeSchemaValidator class implements the ITreeSchemaValidator interface.
+// </summary>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.Serialization.Json;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Forge.DataContracts;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Schema;
+
+
+namespace Microsoft.Forge.TreeWalker.UnitTests.test
+{
+    class TreeSchemaValidator : ITreeSchemaValidator
```

Comment:
>TreeSchemaValidator [](http://example.com/codeflow?start=10&length=19)

The ask is to have a public API for users to validate their schemas.

Let's move this class to src so that it will be included in the Microsoft.Forge.TreeWalker nuget package and available for folks to consume.